KsApi/mutations/inputs/TriggerCapiEventInputTests.swift

```diff
@@ -0,0 +1,25 @@
+@testable import KsApi
+import XCTest
+
+final class TriggerCapiEventInputTests: XCTestCase {
+  func testTriggerCapiEventInputTestsDictionary_WithValue_Success() {
+    let triggerCapiEventInput =
+      TriggerCapiEventInput(
+        projectId: "projId",
+        eventName: "eventName",
+        externalId: "extId",
+        userEmail: "userEmail",
+        appData: GraphAPI.AppDataInput(extinfo: ["i2"]),
+        customData: GraphAPI.CustomDataInput(currency: nil, value: nil)
+      )
+
+    let input = triggerCapiEventInput.toInputDictionary()
+
+    XCTAssertEqual(input["projectId"] as? String, "projId")
+    XCTAssertEqual(input["eventName"] as? String, "eventName")
+    XCTAssertEqual(input["externalId"] as? String, "extId")
+    XCTAssertEqual(input["userEmail"] as? String, "userEmail")
+    XCTAssertNotNil(input["appData"])
+    XCTAssertNotNil(input["customData"])
```

Comment:
Since `GraphAPI.AppDataInput` & `GraphAPI.CustomDataInput` don't conform to Equatable _and_ they're auto generated for us, I opted to just assert that the are not nil here.